### PR TITLE
sys-apps/portage: warn about obsolete enotice in profile.bashrc

### PR DESCRIPTION
**Pull request by floppym**
Bug: https://bugs.gentoo.org/867010

### CODE REVIEW

File: sys-apps/portage/portage-3.0.39-r2.ebuild [hunk 1]
```diff
@@ -272,3 +272,12 @@ pkg_preinst() {
 		elog "change, see https://bugs.gentoo.org/658648."
 	fi
 }
+
+pkg_postinst() {
+	# Remove obsolete "enotice" script, bug #867010
+	local bashrc=${EROOT}/etc/portage/profile/profile.bashrc
```
Comment (thesamesam) [inline sys-apps/portage/portage-3.0.39-r2.ebuild:278]: Let's move it to a file with a random name (mktemp) as there could be content they want to keep in there?

Comment (ionenwks) [reply sys-apps/portage/portage-3.0.39-r2.ebuild:278]: Won't CONFIG_PROTECT come in anyway?

Comment (ionenwks) [reply sys-apps/portage/portage-3.0.39-r2.ebuild:278]: Oh wait, postinst.. nevermind

Comment (ionenwks) [reply sys-apps/portage/portage-3.0.39-r2.ebuild:278]: Maybe it'd make more sense to just warn about it rather than try to remove it.

Comment (floppym) [reply sys-apps/portage/portage-3.0.39-r2.ebuild:278]: Changed it to a an eerror message.

**Review by thesamesam — APPROVED** [review]

**Pull request closed without merging**